lib/components/chat_history/twitch/poll_event.dart

```diff
@@ -0,0 +1,159 @@
+import 'package:flutter/material.dart';
+import 'package:flutter/rendering.dart';
+import 'package:provider/provider.dart';
+import 'package:rtchat/models/messages/twitch/event.dart';
+import 'package:rtchat/models/style.dart';
+import 'dart:math' as math;
+
+List<Widget> getPollsWidget(List<dynamic> choices, List<Widget> children,
+    bool isCompleted, TextStyle baseStyle) {
+  num totalVotes = 0;
+  num totalChannelPointsVotes = 0;
+  num totalBitVotes = 0;
+  num maxVotes = 0;
+
+  for (final entry in choices) {
+    final votes = entry['votes'] ?? 0;
+    final int bitVotes = entry['bit_votes'] ?? 0;
+    final int channelPointVotes = entry['channel_point_votes'] ?? 0;
+    totalVotes += votes;
+    totalChannelPointsVotes += channelPointVotes;
+    totalBitVotes += bitVotes;
+    maxVotes = math.max(maxVotes, votes);
+  }
+
+  for (final entry in choices) {
+    final String choiceName = entry['title'];
+    final votes = entry['votes'] ?? 0;
+
+    // avoid division by zero
+    double percentage = 0;
+    if (totalVotes > 0) {
+      percentage = votes / totalVotes;
+    }
+    if (isCompleted && maxVotes == votes) {
+      children.add(Padding(
+          padding: const EdgeInsets.fromLTRB(0, 0, 0, 8),
```

Comment:
`EdgeInsets.only(bottom: 8)` here and same for the other `fromLTRB`'s with only one side.